### PR TITLE
New Axis vendor function deserialize_attestation

### PR DESCRIPTION
**Pull request by bjornvolcker**
which turns the serialized attestation data, part of the SEI, into an
attestation report.

### CODE REVIEW

File: lib/vendors/axis-communications/sv_vendor_axis_communications.c [hunk 1]
```diff
@@ -250,6 +274,64 @@ verify_and_parse_certificate_chain(sv_vendor_axis_communications_t *self)
...
+ *     - signature (|signature_size| bytes) */
+static svi_rc
+deserialize_attestation(sv_vendor_axis_communications_t *self)
+{
+  assert(self);
+
+  uint8_t *attestation_ptr = (uint8_t *)self->attestation;
+  size_t signature_size = 0;
+
+  if (!self->attestation) return SVI_VENDOR;
```
Comment (lusikamalo) [inline lib/vendors/axis-communications/sv_vendor_axis_communications.c:295]: Maybe we should have the check for (!self->attestation) above row 292 where it's used the first time.

Comment (bjornvolcker) [reply lib/vendors/axis-communications/sv_vendor_axis_communications.c:295]: Great catch!

**Review by lusikamalo — CHANGES_REQUESTED** [review]
No other findings than the single comment I wrote. Waiting for your next patch and I will approve.

**Review by lusikamalo — APPROVED** [review]
OK.

**Pull request merged**